config.js

```diff
@@ -30,5 +30,24 @@ var config = {
       destination : "", // Destination of your trip. Human readable address.
       name : "work", // Name of your destination ex: "work"
       reload_interval : 5 // Number of minutes the information is refreshed
+    },
+    fitbit: {
+        "timeout": 10000,
+        "creds": {
+            "clientID": "CLIENTID",
+            "clientSecret": "SECRET"
+        },
+        "uris": {
+            "authorizationUri": "https://www.fitbit.com",
+            "authorizationPath": "/oauth2/authorize",
+            "tokenUri": "https://api.fitbit.com",
+            "tokenPath": "/oauth2/token"
+        },
```

Comment:
Do these "uris" have to be in the config, or should they be baked into the service as constants?